releases/release-notes.md

```diff
@@ -10,6 +10,7 @@ TiDB 历史版本发布声明如下：
 
 ## 4.0
 
+- [4.0.2](/releases/release-4.0.2.md)
```

Comment:
```suggestion
```
This should not be included in this PR? @uglyengineer

Reply:
yes